packages/mdc-fab/_mixins.scss

```diff
@@ -40,7 +40,10 @@
   width: 56px;
   height: 56px;
   padding: 0;
-  transition: box-shadow 280ms $mdc-animation-standard-curve-timing-function;
+  transition:
+    box-shadow 280ms $mdc-animation-standard-curve-timing-function,
```

Comment:
should this be 270ms?

Reply:
This is existing code and matches `$mdc-elevation-transition-duration`, but I'm not sure why we aren't just using that variable directly, so I'll go fix that :)